pmm/build-dev-container.groovy

```diff
@@ -0,0 +1,34 @@
+Jenkins.instance.getItemByFullName(env.JOB_NAME).description = '''
+Build dev container for dev-latest
+'''
+pipeline {
+    agent {
+        label 'agent-amd64'
+    }
+    environment {
+        DOCKER_IMAGE='perconalab/pmm-server:dev-container'
+    }
+    triggers {
+        upstream upstreamProjects: 'pmm2-server-autobuild', threshold: hudson.model.Result.SUCCESS
+    }
+    stages {
+        stage('Clone repo') {
+            steps {
+                git poll: true,
+                    branch: 'main',
+                    url: "https://github.com/percona/pmm.git"
+            }
+        }
+        stage('Build dev-container') {
+            steps {
+                withCredentials([usernamePassword(credentialsId: 'hub.docker.com', passwordVariable: 'PASS', usernameVariable: 'USER')]) {
+                    sh """
+                        docker login -u "${USER}" -p "${PASS}"
+                    """
+                }
+                sh 'docker build -t $DOCKER_IMAGE -f devcontainer.Dockerfile .'
```

Comment:
Minor: I think it may be a good idea to move `devcontainer.Dockerfile` from pmm repo root to the `/.devcontainer/` directory.

Reply:
@artemgavrilov Yep maybe makes sense. I'll do it in separate PR as part of migration build scripts.